dref/serializers.py

```diff
@@ -14,9 +14,7 @@
 )
 from api.serializers import (
     UserNameSerializer,
```

Comment:
In row 390 there is an unresolved variable, ALLOWED_ASSESSmentREPORT_EXTENSIONS (lowercase typo)